Block/Js.php

```diff
@@ -62,13 +69,15 @@ public function __construct(
         CheckoutSession $checkoutSession,
         CartHelper $cartHelper,
         Bugsnag $bugsnag,
+        Http $request,
         array $data = []
     ) {
         parent::__construct($context, $data);
         $this->configHelper = $configHelper;
         $this->checkoutSession = $checkoutSession;
         $this->cartHelper = $cartHelper;
         $this->bugsnag = $bugsnag;
+        $this->request = $request;
```

Comment:
@johnmccombs1 
Wow! Why did you inject in such way?

Every block have `$this->_request` variable. The variable contain object of class `Magento\Framework\App\Request\Http`.
Even if your block do not have such variable, so every `$context` in block have it - you can inject it
`$this->request = $context->getRequest()` 
(the inherited class is `Magento\Framework\App\Request\Http`).

Please remove this extra injection.